src/fdk_organization_bff/service/org_catalog_service.py

```diff
@@ -266,6 +264,18 @@ async def get_organization_catalog(
         logging.warning("Unable to fetch org info models")
         org_informationmodels = []
 
+    dataset_uris = list(map(lambda ds: ds["dataset"]["value"], org_datasets))
+    org_datasets_scores = {}
+    if len(dataset_uris) > 0:
```

Comment:
`list(map(` hører vel hjemme inni `if`, ettersom det bare skal gjøres dersom det er elementer i `org_datasets`?
```suggestion
    org_datasets_scores = {}
    if len(org_datasets) > 0:
            dataset_uris = [ds["dataset"]["value"] for ds in org_datasets]
```